styleguide/derek/global-components/tracking/tracking.js

```diff
@@ -0,0 +1,124 @@
+        eventLabel: window.location.href,
+        eventValue: '0',
+        eventNonInteraction: 0,
+      });
+    });
+
+    $('.track-ceilingCall').on('click', () => {
+      dataLayer.push({
+        event: 'ga.event',
+        eventCategory: 'Infinity Tracking Triggers',
+        eventAction: 'Call Click CTA - Eyebrow',
+        eventLabel: window.location.href,
+        eventValue: '0',
+        eventNonInteraction: 0,
+      });
+    });
+
+    $('.track-rugEmail').on('click', () => {
+      dataLayer.push({
+        event: 'ga.event',
+        eventCategory: 'Site Submission',
+        eventAction: 'Form Click CTA - Rug',
+        eventLabel: window.location.href,
+        eventValue: '0',
+        eventNonInteraction: 0,
+      });
+    });
+
+    $('.track-secondCTA').on('click', () => {
+      dataLayer.push({
+        event: 'ga.event',
+        eventCategory: 'Site Submission',
+        eventAction: 'Form Click CTA - Secondary CTA',
+        eventLabel: window.location.href,
+        eventValue: '0',
+        eventNonInteraction: 0,
+      });
+    });
+
+    $('.track-rugCall').on('click', () => {
+      dataLayer.push({
+        event: 'ga.event',
+        eventCategory: 'Infinity Tracking Triggers',
+        eventAction: 'Call Click CTA - Rug',
+        eventLabel: window.location.href,
+        eventValue: '0',
+        eventNonInteraction: 0,
+      });
+    });
+
+    $('.track-loginMyRack').on('click', () => {
+      dataLayer.push({
+        event: 'ga.event',
+        eventCategory: 'Login',
+        eventAction: 'Click',
+        eventLabel: 'https://my.rackspace.com/portal/auth/login',
+        eventValue: '0',
+        eventNonInteraction: 0,
+      });
+    });
+
+    $('.track-loginApps').on('click', () => {
+      dataLayer.push({
+        event: 'ga.event',
+        eventCategory: 'Login',
+        eventAction: 'Click',
+        eventLabel: 'https://apps.rackspace.com/',
+        eventValue: '0',
+        eventNonInteraction: 0,
+      });
+    });
+
+    $('.track-loginFaws').on('click', () => {
+      dataLayer.push({
+        event: 'ga.event',
+        eventCategory: 'Login',
+        eventAction: 'Click',
+        eventLabel: 'https://manage.rackspace.com/aws',
+        eventValue: '0',
+        eventNonInteraction: 0,
+      });
+    });
+
+    $('.track-loginCloudOffice').on('click', () => {
+      dataLayer.push({
+        event: 'ga.event',
+        eventCategory: 'Login',
+        eventAction: 'Click',
+        eventLabel: 'https://cp.rackspace.com/',
+        eventValue: '0',
+        eventNonInteraction: 0,
+      });
+    });
+
+    $('.track-loginCloudCp').on('click', () => {
+      dataLayer.push({
+        event: 'ga.event',
+        eventCategory: 'Login',
+        eventAction: 'Click',
+        eventLabel: 'https://mycloud.rackspace.com/',
+        eventValue: '0',
+        eventNonInteraction: 0,
+      });
+    });
+  }
+
+  return {
+    init() {
+      tracking();
+    },
+  };
+}(jQuery));
```

Comment:
Is this based on an older version?  It seems like we're missing quite a few classes, and their respective tests.  I think the most recently added classes to the www version of this are `.track-cta`, which I'm not seeing in here. Can you go double check the www repo, and make sure you're based on the latest version?